docs/software/wifi_hotspot.md

```diff
@@ -2,12 +2,80 @@
 
 ## Overview
 
-- [WiFi HotSpot - Turn your device into a wireless hotspot/access point.](https://dietpi.com/phpbb/viewtopic.php?p=1207#p1207)  
-- [Tor HotSpot - Optional: Routes all WiFi HotSpot traffic through the Tor network.](https://dietpi.com/phpbb/viewtopic.php?p=1529#p1529)  
+- [**WiFi HotSpot - Turn your device into a wireless hotspot/access point**](#wifi-hotspot_1)  
+- [**Tor HotSpot - Optional: Routes all WiFi HotSpot traffic through the Tor network**](#tor-hotspot)  
+
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
+    ```
+    and selected _DietPi-Software_. or launch directly the tool:
+    ```
+    dietpi-software
+    ```
+
+    Choose **Software optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
 
-!!! warning "Documentation in progress..." 
+## WiFi HotSpot
+
+The WiFi HotSpot package turns your device into a wireless hotspot/access point. This allows other wireless devices to connect and share the internet connection.
+
+![DietPi WiFi hotspot WiFi](../assets/images/dietpi-software-wifihotspot-wifihotspot.png){: style="width:550px"}
+
+=== "Requirements"
+
+    The requirements are:
+
+    - 1x Ethernet connection
+    - 1x Supported USB WiFi adapter or onboard WiFi. This may vary depending on device and available WiFi drivers/modules. However, common adapters (e.g.: Realtek) should be fine.
+
+=== "Initial connection credentials"
+
+    Use the following credentials to initially connect devices to your hotspot.
+
+    - SSID = *DietPi-HotSpot*
+    - Access Key = *dietpihotspot*
```

Comment:
What do you think about wrapping all such values into code tags? `DietPi-HotSpot` and `dietpihotspot`, so they don't need to be added to the wordlist. So basically making everything to inline-code what is meant to be copy&pasted to console or input field.

Reply:
Ok, done.  
I think I have to change this on more documents. I will check.